src/agent0/core/hyperdrive/interactive/i_hyperdrive.py

```diff
@@ -148,7 +148,7 @@ def init_agent(
         private_key: str,
         policy: Type[HyperdriveBasePolicy] | None = None,
         policy_config: HyperdriveBasePolicy.Config | None = None,
-    ):
+    ) -> IHyperdriveAgent:
```

Comment:
@slundqui I added this bc I needed it in an earlier iteration of the script, but I don't now. If there's some reason why you didn't do it in the first place lmk and I can delete it.